docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -24,10 +32,41 @@ To create a new user:
 4. Review the new user information and note the temporary password that has been assigned. To send the user an introductory email, select the checkbox.
 5. Press *Done* when complete.
 
+## Users Table
+
+The *Users* table displays a list of users who are authorized to login to the Run:ai platform.
+
+!!! Note
+    SSO users who have not logged into the Run:ai platform will not appear in the table until they do.
+
+The *Users* table contains informtion about users who have logged into the Run:ai Platform. The table includes columns for:
+
+* **User**&mdash;the email and user ID of the user.
+* **Type**&mdash;the type of user (local or SSO).
+* **Last login**&mdash;the last time the user logged into the platform.
+* **Access rule(s)**&mdash;press *VIEW* to view the access rule(s) assigned to the user.
+* **Created by**&mdash;indicates who created the user in the able.
+* **Creation time**&mdash;the timestamp for when the user was created.
+* **Last updated**&mdash;the last time the user information was updated.
```

Comment:
@jasonnovichRunAI "Created by", "Created time", and "Last updated" will be empty for SSO users, do you think we should mention it?

Reply:
@gal-revach We can. It may look something like:
```suggestion
* **Last updated**&mdash;the last time the user information was updated. Not applicable to SSO users.
```
WDYT?

Reply:
@jasonnovichRunAI yes, let's add it to the relevant fields